src/templates/EditorialNewsletter/index.js

```diff
@@ -0,0 +1,251 @@
+import Container from './Container'
+import Cover, { Title, Lead } from './Cover'
+import Paragraph, { Strong, Em, Link, Br } from './Paragraph'
+import Center from './Center'
+import { H2 } from './Headlines'
+import Figure, { Image, Caption, Byline } from './Figure'
+import Blockquote, { BlockquoteText, BlockquoteSource } from './Blockquote'
+import List, { ListItem } from './List'
+
+import {
+  matchType,
+  matchZone,
+  matchHeading,
+  matchParagraph,
+  matchImageParagraph
+} from 'mdast-react-render/lib/utils'
+
+const matchLast = (node, index, parent) => index === parent.children.length - 1
+
+const globalInlines = [
+  {
+    matchMdast: matchType('break'),
+    component: Br,
+    isVoid: true
+  },
+  {
+    matchMdast: matchType('strong'),
+    component: Strong,
+    editorModule: 'mark',
+    editorOptions: {
+      type: 'strong'
+    }
+  }
+]
+
+const link = {
+  matchMdast: matchType('link'),
+  component: Link,
+  props: node => ({
+    title: node.title,
+    href: node.url
+  }),
+  editorModule: 'link'
+}
+
+const paragraph = {
+  matchMdast: matchParagraph,
+  component: Paragraph,
+  editorModule: 'paragraph',
+  editorOptions: {
+    formatButtonText: 'Paragraph'
+  },
+  rules: [
+    ...globalInlines,
+    link,
+    {
+      matchMdast: matchType('emphasis'),
+      component: Em,
+      editorModule: 'mark',
+      editorOptions: {
+        type: 'emphasis'
+      }
+    }
+  ]
+}
+
+const figureCaption = {
+  matchMdast: matchParagraph,
+  component: Caption,
+  editorModule: 'figureCaption',
+  editorOptions: {
+    afterType: 'PARAGRAPH',
+    insertAfterType: 'FIGURE',
+    placeholder: 'Legende'
+  },
+  rules: [
+    {
+      matchMdast: matchType('emphasis'),
+      component: Byline,
+      editorOptions: {
+        placeholder: 'Credit'
+      }
+    },
+    link,
+    ...globalInlines
+  ]
+}
+
+const schema = {
+  emailTemplate: 'newsletter-editorial',
+  repoPrefix: 'newsletter-editorial-',
+  rules: [
+    {
+      matchMdast: matchType('root'),
+      component: Container,
+      editorModule: 'document',
```

Comment:
use `documentPlain` and `COVERFIGURE` instead. `document` & `cover` modules were very specifically coded text over image. We will need to make `documentPlain` work without a title block but shouldn't be too hard. I can help there.

```
  editorModule: 'figure',
  editorOptions: {
    type: 'COVERFIGURE'
  }
```